### DISCUSSION TITLE
Fixate op outputs for development?

### DISCUSSION THREAD

**Discussion by tamis-laan** [dagster-io/dagster#8103] [Q&A]
I'm experimenting with web scraping in Dagster.

I'm first scraping the html of a web page and then parsing it. I would like to freeze/fixate/cache the output of the first nodes that implement the scraping step so I can focus on developing the html parsing step without fetching the website content each time. 

Is this possible?

**Comment by smackesey** [comment]
cc @dpeng817 for memoization advice
